annotation-processing/src/main/java/com/exactpro/epfast/annotation/processing/FastProcessor.java

```diff
@@ -93,6 +92,25 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
     }
 
     private void buildCreatorClasses(FastEnvironment fastEnvironment) throws IOException {
+        for (FastPackageElement fastPackage : fastEnvironment.getFastPackages()) {
+            for (FastTypeElement fastTypeElement : fastPackage.getFastTypes()) {
+                TypeName typeName = new TypeName(
+                    String.format("com.exactpro.epfast.annotation.internal.%s.%sFieldSetter",
+                        FastPackageNameEncoder.encode(fastPackage.getPackageName()),
+                        fastTypeElement.getTypeName())
```

Comment:
We don't put any restrictions on FAST type name. Doing like this may result in invalid Java name. Create another ticket in Redmine to create valid Java name for FAST type name